packages/flutter/test/widgets/custom_painter_test.dart

```diff
@@ -401,6 +401,9 @@ void _defineTests() {
         case SemanticsAction.showOnScreen:
         case SemanticsAction.tap:
           semanticsOwner.performAction(expectedId, action);
+        // Add the default case temporarily to unblock engine roll.
```

Comment:
Can you please make these TODOs in the style I did here? https://github.com/flutter/flutter/pull/149274/files

It will make it easier for me to undo later.